src/id3/v2/frame/mod.rs

```diff
@@ -289,10 +321,35 @@ impl FrameValue {
 			FrameValue::Picture(attached_picture) => attached_picture.as_bytes(Id3v2Version::V4)?,
 			FrameValue::Popularimeter(popularimeter) => popularimeter.as_bytes(),
 			FrameValue::KeyValue(content) => content.as_bytes(),
-			FrameValue::Binary(binary) => binary.clone(),
+			FrameValue::RelativeVolumeAdjustment(frame) => frame.as_bytes(),
 			FrameValue::UniqueFileIdentifier(frame) => frame.as_bytes(),
+			FrameValue::Ownership(frame) => frame.as_bytes()?,
+			FrameValue::EventTimingCodes(frame) => frame.as_bytes(),
+			FrameValue::Private(frame) => frame.as_bytes(),
+			FrameValue::Binary(binary) => binary.clone(),
 		})
 	}
+
+	/// Used for errors in write::frame::verify_frame
+	pub(super) fn name(&self) -> &'static str {
```

Comment:
I use `strum` for this purpose: https://docs.rs/strum/latest/strum/trait.VariantNames.html

Reply:
Oh yeah, looks like they have some useful derives. There are probably a few places they could be used. I might look into it eventually.